trivia.py

```diff
@@ -229,14 +230,14 @@ def _help(self, args, user, channel):
         try:
             self._admins.index(user)
         except:
-            self._cmsg(user, "I'm nameless's trivia bot.")
+            self._cmsg(user, "I'm %s's trivia bot." % config.OWNER)
```

Comment:
Hi Andy,

Could you use `I'm {}'s trivia bot.".format(config.OWNER)` instead? The % syntax is old and the format() method is preferred moving forward in python.